openapi.yaml

```diff
@@ -143,374 +82,417 @@ paths:
+        '404':
+          $ref: '#/responses/404'
+  /persons/{osuId}/meal-plans:
     get:
       description: Get meal plans for a person
       tags:
         - meal-plans
       operationId: getMealPlans
       parameters:
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/osuId'
+        - name: mealPlan
+          in: query
+          type: string
+          required: false
+          description: Filter results by meal plan name.
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/MealPlansResultObject'
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/meal-plans/{mealPlanID}:
+            $ref: '#/definitions/MealPlanSetResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/meal-plans/{mealPlanId}:
     get:
       description: Get a specific meal plan for a person
       tags:
         - meal-plans
-      operationId: getMealPlanByID
+      operationId: getMealPlanById
       parameters:
-        - in: path
-          name: mealPlanID
-          description: ID of meal plan
-          required: true
-          type: string
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/osuId'
+        - $ref: '#/parameters/mealPlanId'
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/MealPlanResultObject'
-        404:
-          description: Person or meal plan not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/addresses:
+            $ref: '#/definitions/MealPlanResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/address:
     get:
       description: Get addresses for a person
       tags:
         - addresses
       operationId: getAddresses
       parameters:
+        - $ref: '#/parameters/osuId'
+        - name: city
+          in: query
+          type: string
+          required: false
+          description: Filter results by city.
+        - name: county
+          in: query
+          type: string
+          required: false
+          description: Filter results by county.
+        - name: state
+          in: query
+          type: string
+          required: false
+          description: Filter results by state.
+        - name: stateCode
+          in: query
+          type: string
+          required: false
+          description: Filter results by state code.
+        - name: nation
+          in: query
+          type: string
+          required: false
+          description: Filter results by nation.
+        - name: nationCode
```

Comment:
Can you add enum list to those filters by codes?

Reply:
Do we want to put all ~240 nation codes on there?

Reply:
Shoot, hmm, I guess I personally will lean toward adding it to the API spec for the clearness. But I agree it's a giant list, so let's discuss this with other in the team meet and see how they feel.